### PR TITLE
Added requster pays option to germline and somatic CNV WDLs

### LINKED ISSUE #6829: CNV WDL Requester Pays
## Feature request

Somatic/Germline CNV WDL

### Description
Many of the users on Terra that work with the GATK workflows get stuck working with requester-pays data. (e.g. [forum post](https://gatk.broadinstitute.org/hc/en-us/community/posts/360067820111/comments/360011055131)). This request asks that the workflow include the requester pays option like the [pathseq](https://github.com/broadinstitute/gatk/blob/5e5747b76fa98a3b6731dbc328e292fa941f269b/scripts/pathseq/wdl/pathseq_pipeline.wdl#L83) workflow.

### PR DESCRIPTION
**Pull request by asmirnov239**
Closes #6829
@mwalker174 Could you please review? 

I only added `gcs_project_for_requester_pays` input to tasks that need access to BAMs. Do we also need it for the ones that require reference such as `PreprocessIntervals`?

### CODE REVIEW

File: scripts/cnv_wdl/somatic/cnv_somatic_panel_workflow.wdl [hunk 1]
```diff
@@ -55,6 +55,9 @@ workflow CNVSomaticPanelWorkflow {
       File? gatk4_jar_override
       Int? preemptible_attempts
 
+      # Required if BAM/CRAM is in a requester pays bucket
+      String? gcs_project_for_requester_pays
```
Comment (mwalker174) [inline scripts/cnv_wdl/somatic/cnv_somatic_panel_workflow.wdl:59]: Add this to `CollectCounts` input

**Review by mwalker174 — CHANGES_REQUESTED** [review]
Thanks @asmirnov239, just a few corrections here.

**Comment by ldgauthier** [comment]
@mwalker174 Since Andrey is out, I addressed your comments and I think I caught a couple more missing inputs.

**Review by mwalker174 — APPROVED** [review]
Looks good to me!

**Pull request merged**